k8s/upgrade/src/bin/upgrade-job/helm/chart.rs

```diff
@@ -534,13 +565,20 @@ impl CsiImage {
 #[derive(Deserialize)]
 struct CsiNode {
     nvme: CsiNodeNvme,
+    #[serde(default, rename(deserialize = "pluginMounthPath"))]
```

Comment:
`pluginMountPath` --> once @abhilashshetty04's PR gets merged.

Reply:
No, that's got to be `pluginMounthPath`. The intention here is to collect the MounthPath value and set it to MountPath on 2.6.0.